projects/tutorial_board_adc/src/main.c

```diff
@@ -1,4 +1,4 @@
-#include "adc.h"
+#include "../../../libraries/ms-common/inc/adc.h"
```

Comment:
This change should not be included as part of the PR. 
When we make PR's we are saying "I would like these changes to be added to the codebase and for everyone else to see it once it has been merged"

Though this is just the homework, its still best to keep PR's clean